js/slider.js

```diff
@@ -0,0 +1,86 @@
+window.onload = () => {
+    nextSlide();
+    slideRotation();
+};
+
+var nextButton = document.querySelector(".slider-button-next");
+nextButton.addEventListener("click", nextSlide);
+
+var prevButton = document.querySelector(".slider-button-prev");
+prevButton.addEventListener("click", prevSlide);
+
+var arrowButtons = document.querySelectorAll(".slider-button");
+arrowButtons.forEach(button => {button.addEventListener("click", cancelSlideRotation)});
+
+var dots = document.querySelectorAll(".slider-dots-element");
+
+var thisAd = 0;
+var slides = [
+  {
```

Comment:
U mnie slajdy się nie wczytują przy dwóch kropkach w urlu przed slashem, pokazują się po usunięciu jednej kropki w każdym linku.

Reply:
Faktycznie. Nie do końca rozumiem dlaczego. Na LiveSerwer działało poprawnie. Co więcej, poprawny zapis wyjścia z obecnego katalogu, to właśnie dwie kropki. Przynajmniej jeśli podpinasz pliki w css. Jeśli zerkniesz np. w team w miejsce gdzie podpinamy urle z naszymi zdjęciami, to tam mamy dwie kropki. Niemniej zmieniam zgodnie z sugestią.